app/src/substrate/substrate_types.h

```diff
@@ -178,6 +224,42 @@ typedef struct {
     pd_AccountId_t id;
 } pd_TuplePerbillAccountId_t;
 
+typedef struct {
+    uint64_t _len;
+    const uint8_t* _ptr;
+    uint64_t _lenBuffer;
+} pd_VecBytes_t;
+
+typedef struct {
+    uint64_t _len;
+    const uint8_t* _ptr;
+    uint64_t _lenBuffer;
+} pd_VecTupleDataData_t;
```

Comment:
Not critical, but why the repeated DataData?

Reply:
This represents the type `BoundedVec<(Data, Data)>`, so the name is correct